hack/gen-flavors.sh

```diff
@@ -28,6 +28,15 @@ flavors_dir="${REPO_ROOT}/templates/flavors/"
 ci_dir="${REPO_ROOT}/templates/test/ci/"
 dev_dir="${REPO_ROOT}/templates/test/dev/"
 
+# NOTE: On macOS, the kustomize commands run by xargs that use the -I
```

Comment:
I added this comment mostly to be breadcrumbs for the next time we hit this since I think it might be difficult to track down this PR otherwise.